core/block/editor/smartblock/links.go

```diff
@@ -23,6 +23,10 @@ func (sb *smartBlock) updateBackLinks(s *state.State) {
 		log.With("objectID", sb.Id()).Errorf("failed to get inbound links from object store: %s", err)
 		return
 	}
+	if len(backLinks) != 0 && sb.space != nil && sb.Id() != sb.currentProfileId {
```

Comment:
`sb.space != nil`
This check seems redundant? 🤔

Reply:
We have decided to not index widget links at all